RecoLocalMuon/RPCRecHit/src/RPCCluster.cc

```diff
@@ -1,71 +1,84 @@
 #include "RPCCluster.h"
 #include <iostream>
 #include <fstream>
+#include <cmath>
 
 using namespace std;
 
-RPCCluster::RPCCluster() : fstrip(0), lstrip(0), bunchx(0)
+RPCCluster::RPCCluster():
+  fstrip(0), lstrip(0), bunchx(0),
+  sumTime(0), sumTime2(0), nTime(0), sumY(0), sumY2(0), nY(0)
 {
 }
 
-RPCCluster::RPCCluster(int fs, int ls, int bx) : 
-  fstrip(fs), lstrip(ls), bunchx(bx)
+RPCCluster::RPCCluster(int fs, int ls, int bx) :
+  fstrip(fs), lstrip(ls), bunchx(bx),
+  sumTime(0), sumTime2(0), nTime(0), sumY(0), sumY2(0), nY(0)
 {
 }
 
-RPCCluster::~RPCCluster()
-{
-}
+RPCCluster::~RPCCluster() {}
+
+int RPCCluster::firstStrip() const { return fstrip; }
+int RPCCluster::lastStrip() const { return lstrip; }
+int RPCCluster::clusterSize() const { return -(fstrip-lstrip)+1; }
```

Comment:
Just because the formula have been there, I can just guess that this is for the code readability.